rest_configuration.cpp

```diff
@@ -1022,12 +1020,10 @@ void DeRestPluginPrivate::configToMap(const ApiRequest &req, QVariantMap &map)
     swupdate2["bridge"] = bridge;
     swupdate2["checkforupdate"] = false;
     swupdate2["state"] = gwSwUpdateState;
-    swupdate2["install"] = false;
     autoinstall["updatetime"] = "";
     autoinstall["on"] = false;
     swupdate2["autoinstall"] = autoinstall;
     swupdate2["lastchange"] = "";
-    swupdate2["lastinstall"] = "";

```

Comment:
"install" and "lastinstall" are present in the Hue API why remove them?

Reply:
The Hue API documentation is not very clear on this, but from the actual bridge it looks like:
- "install" is a write only item
- "lastinstall" is only part of the swupdate2 state for the bridge subobject